tests/rules/test_anchors.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (C) 2022 Sergei Mikhailov
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from tests.common import RuleTestCase
+
+
+NORMAL_ANCHOR = '''---
+key: &keyanchor a
+otherkey: *keyanchor
+'''
+
+NORMAL_ANCHOR_NO_DOC_START = '''---
+key: &keyanchor a
+otherkey: *keyanchor
+'''
+
+DUPLICATED_ANCHOR = '''---
+key1: &keyanchor a
+key2: &keyanchor b
+otherkey: *keyanchor
+'''
```

Comment:
Please add a test for anchors duplicated 3 times. Latest version of the PR (commit 542e05f) seems to have this bug!